var/spack/repos/builtin/packages/mapl/package.py

```diff
@@ -34,11 +34,12 @@ class Mapl(CMakePackage):
     version("2.7.1", sha256="8239fdbebd2caa47a232c24927f7a91196704e35c8b7909e1bbbefccf0647ea6")
 
     # Versions later than 3.14 remove FindESMF.cmake
-    # from ESMA_CMake. This works with mapl@2.22.0:
+    # from ESMA_CMake. This works with mapl@2.22.0
+    # and latest Apple M1 processors.
```

Comment:
@climbfuji Maybe we should say:
```suggestion
    # and latest Apple M1 and M2 processors.
```